src/main/resources/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM/help-sshCredentialsId.html

```diff
@@ -0,0 +1,11 @@
+<div>
+    <p>If specified, Jenkins will use these credentials to check out the source code for builds. If no SSH credentials
+        are specified, Jenkins will use the Basic Credentials instead.</p>
+
+    <p>To provide Jenkins with SSH credentials, you can:</p>
+    <ul>
+        <li>choose credentials from the list</li>
+        <li>add credentials as a <strong>SSH Username with private key</strong> (the username must be "git")</li>
+        <li>Leave this empty to use the credentials if they exist.</li>
```

Comment:
I'm confused over this last entry? If I leave this empty it will use the `Basic Credentials` yeah? So this line is under "to provide Jenkins with **SSH** credentials". I think it needs to be either removed, or updated to be clearer, depending on what this means.

Reply:
Thanks for catching. Given the element belongs to the list "to provide Jenkins with SSH credentials", probably better if it's removed entirely.